readme.md

```diff
@@ -3,70 +3,261 @@
 * Explain what ReactJS is.
 * Explain the component model of web development.
 * Create and render React components in the browser.
+* Pass in data to a React component via `props`.
 * Nest React components.
-* Modify the state of a React component through events.
+* Modify the `state` of a React component through events.
 
 ## What is ReactJS?
 
-### Some History (10 / 10)
+React is a library used to craft modern day UI and create views for a the front-end in web, client and native applications.
+
+> **Selling Point:** By modeling small compatible components that focus on just rendering a view, we as developers can move business logic out of the DOM, and therefore improve our app's performance, maintainability, modularity, and readability.
+
+### Some History
 The first thing most people hear about React is "Facebook uses it."
-* Recently went open-source.
-* Born out of their frustration with the traditional MVC model and how.
+* First used by Facebook in 2011. Then Instagram in 2012.
+* Went open source in May 2013.
+
+React was born out of Facebook's frustration with the traditional MVC model and how...
   * Re-rendering something meant re-rendering everything (or just a lot).
   * That had negative implications on processing power and ultimately user experience, which at times became glitchy and laggy.
-* So they found a creative way to tackle this that flips the Javascript MVC on its head and, in a way, shakes up everything you guys have learned about front-end web development in class so far.
 
-Here's a taste of what React is all about: [https://www.youtube.com/watch?v=KVZ-P-ZI6W4&feature=youtu.be&t=510](https://www.youtube.com/watch?v=KVZ-P-ZI6W4&feature=youtu.be&t=510)  
+> If you want to get a taste of what React's all about, [here's an introduction from React.js Conf 2015.](https://www.youtube.com/watch?v=KVZ-P-ZI6W4&feature=youtu.be&t=510)  
+
+### React in the MVC
+
+**React only concerns our "Views".**
+
+<details>
+  <summary>What is a "view" mean in front-end Javascript?</summary>
+
+  > Visual representations of our models - not the entire page.
+
+</details>
+
+React can coexist with Models and Controllers. The user can set those up however they see fit.
+* This means that React can also coexist with other Javascript frameworks. Let them handle the models and controllers, and have React sort out the views.
+
+# Initial Setup
+
+> **NOTE:** We're about to go through a lot of set-up that may not make 100% sense after going through it once. Don't worry if you don't understand the "Why?" quite yet. You will get more practice with this during the second React lesson.
```

Comment:
Hesitant to include this because not sure how much more in depth we will go into it in React2